asciidoc-confluence-publisher-cli/src/it/java/org/sahli/asciidoc/confluence/publisher/cli/ArgumentsParserTest.java

```diff
@@ -100,4 +100,23 @@ public void optionalJsonArgument_valuePresent_returnsParsedValue() {
         assertThat(value.get().get("attr1"), is("val1"));
         assertThat(value.get().get("attr2"), is("val2"));
     }
+
+    @Test
+    public void optionalBooleanArgument_valuePresent_returnsParsedValue() {
```

Comment:
These two test are hard to read for me, as it is not clear, whether the actual value is coming from the optional, or from the fallback. I suggest to 

a) test on the actual optional return value, without using the default
b) adding one single test per case, not mixing different cases (if a test fails, it is not immediately clear from the broken test name what case is no longer working - I like the semantics of the test cases as a specification)

e.g.

```
    @Test
    public void optionalBooleanArgument_keyWithoutValue_returnsOptionalOfTrue() {
        // arrange
        String[] args = {"keyWithoutValue"};
        ArgumentsParser parser = new ArgumentsParser();

        // act
        Optional<Boolean> value = parser.optionalBooleanArgument("keyWithoutValue", args);

        // assert
        assertTrue(value.isPresent());
        assertThat(value.get(), is(true));
    }

    @Test
    public void optionalBooleanArgument_keyWithTrueValue_returnsOptionalOfTrue() {
        // arrange
        String[] args = {"keyWithValue=true"};
        ArgumentsParser parser = new ArgumentsParser();

        // act
        Optional<Boolean> value = parser.optionalBooleanArgument("keyWithValue", args);

        // assert
        assertTrue(value.isPresent());
        assertThat(value.get(), is(true));
    }

    @Test
    public void optionalBooleanArgument_keyWithFalseValue_returnsOptionalOfFalse() {
        // arrange
        String[] args = {"keyWithValue=false"};
        ArgumentsParser parser = new ArgumentsParser();

        // act
        Optional<Boolean> value = parser.optionalBooleanArgument("keyWithValue", args);

        // assert
        assertTrue(value.isPresent());
        assertThat(value.get(), is(false));
    }

    @Test
    public void optionalBooleanArgument_argumentNotPresent_returnsEmpty() {
        // arrange
        String[] args = {};
        ArgumentsParser parser = new ArgumentsParser();

        // act
        Optional<Boolean> value = parser.optionalBooleanArgument("key", args);

        // assert
        assertFalse(value.isPresent());
    }
```